### PR TITLE
feat: Add rock paper scissors

### PR DESCRIPTION
**Pull request by moshoodabdullahi**
# This PR adds rock paper scissors project is built using semantics HTML, CSS properties and JS
Rock paper scissors is a game played between two people, in which each player simultaneously forms one of three shapes with an outstretched hand, and this project made it possible for only one player to be able to play against the computer.
This project following the [Frontend Mentor assignment link](https://www.frontendmentor.io/challenges/rock-paper-scissors-game-pTgwgvgH) is built with 

- HTML5 elements
  - Semantic elements such as `section`, `header` and so on,
- CSS properties
  - CSS Positioning `fixed`, `relative`,
  - CSS Variables for reusable CSS values,
  - Typography such as `font-size`, `font-weight` and so on,
  - CSS Background for page aesthetic using `background-color`,`background`,
  - CSS Media Query for page responsiveness using `@media` rule,
  - CSS flexbox such as `justify-content`, `align-items` and so on.
- JavaScript
  - Documents such as `querySelector`,
  - Block Scope such as `let`, `const` and so on,
  - Built-in Methods such as `Math.random()`, `Math.floor()` and so on,
  - Linear Data Structure such as `an array`,
  - DOM property such as `textContent`,
  - Conditional Statements such as `if...else statement`,
  - Addition Assignment such as `+=`,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,9 +102,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration, resources/assets used
-- etc
+- [The Odin Project](https://www.theodinproject.com/dashboard)
```
Comment (Yinkusadel) [inline README.md]: the link can be 
https://www.theodinproject.com/lessons/foundations-rock-paper-scissors
instead

**Review by Yinkusadel — COMMENTED** [review]
thats all for me ,rest is fine

**Comment by Yinkusadel** [comment]
### Nice work 
- #### Though making changes in the responsiveness of the rules display will make the UI much better,the rock icon in the rules pops out also the rules display overflows at most screen sizes
- #### when the game is played the color to display should be the color representing the option choosed .E.G red for rock,blue for paper,yellow for scissors

**Fix these and you should be good to go,  Nice work so far👏👏👏**

**Comment by Yinkusadel** [comment]
image
your rules still looks like this on desktop view

**Comment by Yinkusadel** [comment]
image
looks like this on mobile view

**Comment by Yinkusadel** [comment]
hdimage

**Comment by Yinkusadel** [comment]
image
Also this doesnt look well for the obile view as the rules button should now be centered

**Comment by Yinkusadel** [comment]
Also when the games is played the color representing each icon should be what is shown 

here for instance its a tie both has scissors but one is blue one is red ,scissors should be yellow and on both sides

**Comment by Yinkusadel** [comment]
image
your rules display in mobile menu isnt looking too well,it should be centered and the close icon button sould be close to the buttom

**Comment by Yinkusadel** [comment]
image
when played what player picked and what computer picked should display below their images the you win and lose button should be attended to also they should be under the icons not just between them

**Review by teckcipher — APPROVED** [review]
Please add a more descriptive PR description before merging.

**Pull request merged**